Draft Agenda: Collaborator Summit Montreal 2019 (#201)
openjs-foundation/summit: 2019-12-Montreal/agenda.md

# Collaborator Summit Montreal 2019 - December 13th and 14th 🇨🇦

This is a live document that will be constantly updated to match the proposed agenda.

A collaborator summit is a mix of presentations and interactive sessions lead by the Teams, Working Groups, and Projects that participate in the summit. For interactive sessions, our time is best served if a the team/working groups/project participate in good numbers (quorum etc).

> __We can change everything__, so if you think a better structure will suit us better, let us know in #201, or just send a PR!

Reviewer: ```suggestion
__We can change everything__, so if you think a better structure will suit us better, let us know in [#202](https://github.com/openjs-foundation/summit/issues/202), or just send a PR!
```

It would be fantastic if each one of the breakouts is facilitated by a person who will be responsible to "run" the breakout and do a quick presentation at the end. This person would also set outcomes from the breakout to  optimally leverage our time together. There is a facilitator guide available [here](https://github.com/nodejs/summit/blob/master/SESSION_FACILITATOR_GUIDE.md).

✨ We're excited for Node.js' 6th round of collaboration IRL! Even better, we're excited that folks from the JSF will be joining us for the first time! ✨

## Montreal Convention Centre, Montreal
[Palais des congrès de Montréal, Montréal, Canada](https://goo.gl/maps/UAkCWyVxGhwp3BDD7)

### Timings: 
  09:00 - 18:00

### Room Setup - both days
TBA

## Collaborator Summit Day One: 13th December, 2019

### Agenda
TBA

> _To speak/present a topic/do a sesssion, please open a issue in repo. Check https://github.com/openjs-foundation/summit/issues/184 for more details.

Reviewer: ```suggestion
_To speak/present a topic/do a sesssion, please open a issue in repo. Check https://github.com/openjs-foundation/summit/issues/184 for more details._
```

> ## Collaborator Summit Day Two: 14st December, 2019

Reviewer 2: ```suggestion
## Collaborator Summit Day Two: 14th December, 2019
```

### Agenda
TBA
_To speak/present a topic/do a sesssion, please open a issue in repo. Check https://github.com/openjs-foundation/summit/issues/184 for more details.

## Organizers

- Tracy Lee ([@ladyleet](https://github.com/ladyleet))
- Jory ([@jorydotcom](https://github.com/jorydotcom))
- TBA

## Attendance

> Anyone can come, but we will not be explicitly onboarding at Collab Summit Berlin. Conversations can move fast as working groups have a lot of context.

Reviewer 2: ```suggestion
Anyone can come, but we will not be explicitly onboarding at Collab Summit Montreal. Conversations can move fast as working groups have a lot of context.
```

Project maintainers who do not follow or enforce the Code of Conduct in good faith may face temporary or permanent repercussions as determined by other members of the project's leadership.

To learn more about the process of how moderators enforce the Code of Conduct, review the [Moderation Policy](https://github.com/nodejs/admin/blob/master/Moderation-Policy.md#moderation-policy) and [Requesting Moderation](https://github.com/nodejs/admin/blob/master/Moderation-Policy.md#requesting-moderation).

### Attribution

This Code of Conduct is adapted from the [Contributor Covenant](https://www.contributor-covenant.org/), version 1.4, available [HERE](https://www.contributor-covenant.org/version/1/4/)

### Appendix A: The report@nodejs.org Alias

Note: At the current time, all email sent to the [report@nodejs.org](mailto:report@nodejs.org) email alias is copied to all members of the [Node.js Moderation Team](https://github.com/nodejs/admin/blob/master/Moderation-Policy.md#current-members).

------------------------

> ✨ Super stoked to see everyone in May! ✨

Reviewer 3: ```suggestion
✨ Super stoked to see everyone in December! ✨
```
